portal/templates/redesign/teach_new/materials_new.html

```diff
@@ -81,15 +83,15 @@ <h1 class="banner__text--primary">Teaching Resources</h1>
             <div class="row">
                 <div class="col-sm-6">
                     <h4>Key Stage 1, and Lower Key Stage 2</h4>
-                    <p class="body-text"><a>Introduction to Coding Guide</a></p>
-                    <p class="body-text"><a>Levels Guide (1-50)</a></p>
-                    <p class="body-text"><a>Glossary</a></p>
+                    <p class="body-text"><a id="pdf_link" href= "{% url 'materials_viewer_new' pdf_name='introduction_to_coding' %}">Introduction to Coding Guide</a></p>
```

Comment:
The link actually does go to a web page. It's a page which contains the usual header and sub nav, and then a PDF viewer within the body.
James has also indeed included some zip/pdf icons in his wireframe, they need to be added to this page :)